### PR TITLE
#451 : Add logic to support more specifiers for import-name

### PR DESCRIPTION
**Pull request by MonicB14**
This PR fixes [#451](https://github.com/Microsoft/tslint-microsoft-contrib/issues/451)

 - Updated `import-name` validation logic to include more specifiers as replacement-keys
 - Added test case

### CODE REVIEW

File: src/importNameRule.ts [hunk 1]
```diff
@@ -84,15 +84,15 @@ class ImportNameRuleWalker extends ErrorTolerantWalker {
     }
 
     private validateImport(node: ts.ImportEqualsDeclaration | ts.ImportDeclaration, importedName: string, moduleName: string): void {
-        moduleName = moduleName.replace(/.*\//, ''); // chop off the path
-        moduleName = this.makeCamelCase(moduleName);
-        if (this.isImportNameValid(importedName, moduleName) === false) {
-            const message: string = `Misnamed import. Import should be named '${moduleName}' but found '${importedName}'`;
+        let expectedImportedName = moduleName.replace(/.*\//, ''); // chop off the path
```
Comment (MonicB14) [inline src/importNameRule.ts]: I changed the variable name `moduleName` to `expectedModuleName` as it was confusing.<br> Also the primary reason for new variable was to make sure that we aren't changing the input we get which the code was doing earlier.

File: tslint-warnings.csv [hunk 2]
```diff
@@ -218,7 +218,7 @@ CWE 710 - Coding Standards Violation"
 no-void-expression,Requires expressions of type `void` to appear in statement position.,TSLINT8CA59S,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,,
 no-with-statement,Do not use with statements. Assign the item to a new variable instead,TSLINT185D93L,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,"398, 710","CWE 398 - Indicator of Poor Code Quality
 CWE 710 - Coding Standards Violation"
-non-literal-fs-path,Detect calls to fs functions with a non literal filepath,TSLINTJNBF48,tslint,SDL,Error,Critical,Mandatory,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,22,"CWE 22 - undefined"
+non-literal-fs-path,Detect calls to fs functions with a non literal filepath,TSLINTJNBF48,tslint,SDL,Error,Critical,Mandatory,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,22,"CWE 22 - Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal')"
```
Comment (JoshuaKGoldberg) [inline tslint-warnings.csv]: Is this related? If not, could you please undo this change? It might have slipped in from another pull request...

Comment (JoshuaKGoldberg) [reply tslint-warnings.csv]: Ahh, the rest of the code looks good - this is fine 🙂

Comment (MonicB14) [reply tslint-warnings.csv]: Wow this was fast, I was about to make a commit for updating documentation, but then I saw PR was merged 😄  Thanks @JoshuaKGoldberg
